### ISSUE TITLE
json-page layout needs logic for empty tagline

### ISSUE THREAD

**Issue by greensteph** [GSA/challenges-and-prizes#520]
When a historical challenge does not have a tagline an empty H3 tag is output in the html causing a pa11y error.
